rpc/core/tx.go

```diff
@@ -17,6 +17,13 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
+const (
+	txStatusUnknown   string = "UNKNOWN"
```

Comment:
[non-blocking][microscopic nit]
probably document the cases where the status is unknown